packages/react-components/react-components/src/Concepts/Icons/ReactIconGrid.tsx

```diff
@@ -11,39 +11,40 @@ const useStyles = makeStyles({
     display: 'flex',
     flexWrap: 'wrap',
     justifyContent: ' flex-start',
+    backgroundColor: '#F7F7F7',
 
-    '> span': {
-      alignItems: 'center',
-      color: '#3b3a39',
-      display: 'flex',
-      flexDirection: 'column',
-      height: '80px',
-      justifyContent: 'space-around',
-      ...shorthands.padding(0),
-      width: '80px',
-      ...shorthands.overflow('hidden'),
+    '> div': {
+      ...shorthands.margin('8px'),
+      width: '25%',
```

Comment:
This layout isn't giving you what you expect here. It's not a 4 up layout. 

![image](https://user-images.githubusercontent.com/1434956/186262510-48f4ce60-219d-4bcf-9150-c18564d532ea.png)


Consider using grid + grid gap instead.

Reply:
was about to say this, you could do something like this: https://codepen.io/sopranopillow/pen/PoRgbRL